picamera2/picamera2.py

```diff
@@ -128,7 +129,7 @@ def asynchronous(self) -> bool:
 
     @property
     def camera_properties(self) -> dict:
-        return {} if self.camera is None else self.camera.properties
+        return {} if self.camera is None else self.camera_properties
```

Comment:
Does this work? I was expecting it to return `self.camera_properties_` rather than `self.camera_properties`, which feels like it might be an infinite recursion once `self.camera` is not `None`.